config.ts

```diff
@@ -182,3 +226,4 @@ export const website_config = config.website_config;
 export const footer_config = config.footer_config;
 export const page_list = config.page_list;
 export const home_page = config.home_page;
+export const events = config.events;
```

Comment:
> just a general comment about events. I think in a year or two, if we maintain all past events in the config, it will result in the config being slightly bloated. At some point down the line lets consider moving past events to its own file when finished

I know this is a future note my thoughts here are that i think we should wait and see what happens as you mention, and having `config.ts` be a seperate file actually makes it really easy for us to just merge two seperate config files here if needed, whenever that comes up.